backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -442,10 +532,19 @@ def _export_attrs(
         self,
         deployment: client.V1Deployment,
         service: client.V1Service,
+        ports: dict[str, int],
     ) -> dict[str, t.Any]:
+
+        if "rdp" in ports:
+            port = {ports["rdp"]}
+        elif "http" in ports:
+            port = {ports["http"]}
+        else:
+            port = {ports.values()}
+
         return {
             "id": deployment.to_dict()["metadata"]["name"],
-            "ports": {3389},
+            "ports": port,
```

Comment:
What is the purpose of this? As far as I can see `"ports"` is only used in one place to get the rdp port. So why do we need the other things here?